src/cli/commands/test/iac-local-execution/index.ts

```diff
@@ -62,6 +64,21 @@ export async function test(
       options,
       iacOrgSettings.meta,
     );
+
+    try {
+      await trackUsage(formattedResults);
+    } catch (e) {
+      if (e instanceof TestLimitReachedError) {
+        throw e;
+      }
+
+      // If something has gone wrong, err on the side of allowing the user to
+      // run their tests.
+      // TODO propagate error
+      console.log(e);
```

Comment:
@rontalx @ipapast what do you reckon about just logging these errors, and if so, where and how? From Ben on https://snyksec.atlassian.net/browse/CC-972:

> We shouldn’t block the local exec from running if we can’t report the test data. The test data helps us understand usage and if we chose to enforce test limits would help with that, but i’d optimise for the developer experience of being able to run a test

This produces output that looks like this:

```
CustomError: An error occurred while attempting to track test usage: {"code":429,"message":"Test limit reached!","userMessage":"Test limit reached!","error":"Test limit reached!","errorRef":"c38824c4-539e-412a-98c7-7bfc994831e2"}
    at trackUsage (/Users/craig/workspace/github.com/snyk/snyk/src/cli/commands/test/iac-local-execution/usage-tracking.ts:34:13)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at Object.trackUsage (/Users/craig/workspace/github.com/snyk/snyk/src/cli/commands/test/iac-local-execution/measurable-methods.ts:29:25)
    at test (/Users/craig/workspace/github.com/snyk/snyk/src/cli/commands/test/iac-local-execution/index.ts:69:7)
    at Object.localTest (/Users/craig/workspace/github.com/snyk/snyk/src/cli/commands/test/iac-local-execution/measurable-methods.ts:29:25)
    at test (/Users/craig/workspace/github.com/snyk/snyk/src/cli/commands/test/index.ts:89:39)
    at runCommand (/Users/craig/workspace/github.com/snyk/snyk/src/cli/index.ts:57:25)
    at main (/Users/craig/workspace/github.com/snyk/snyk/src/cli/index.ts:313:11) {
  code: undefined,
  strCode: undefined,
  innerError: undefined,
  userMessage: undefined
}

Testing k8s.yaml...


Infrastructure as code issues:
  ✗ Container is running in privileged mode [High Severity] [SNYK-CC-K8S-1] in Deployment
...
```

Reply:
Hmm I like the idea of not failing the run, but how do we beautify this? :D

Reply:
Should we just console log the message? 
```suggestion
      console.log(e.userMessage);
```

Reply:
@rontalx and I paired a bit, and he suggested that the user shouldn't really care about these non-fatal errors. I agreed, so we removed the log. I think we can rely on server-side monitoring and alerting on this endpoint.